apps/io-public/extensions.csproj

```
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

I would align this to be `net6.0`. azure functions runtime requires at least this .NET runtime version. however I think this file is completely ignored but I may be wrong
```suggestion
    <TargetFramework>net6.0</TargetFramework>
```

---

In doubt, I'm going to commit the suggestion both for consistency with the other functions and for documentation purpose.